components/src/core/components/Skeleton/skeleton.scss

```diff
@@ -0,0 +1,43 @@
+@import '../../../styles';
+@import 'variables';
+
+.oxd-skeleton {
+  width: 100%;
+  height: 1.2rem;
+  overflow: hidden;
+  position: relative;
+  margin-bottom: 5px;
+  border-radius: $oxd-border-radius;
+  background-color: $skeleton-background-color;
+
+  &.--animate {
+    &::after {
+      content: '';
+      position: absolute;
+      top: 0;
+      left: 0;
+      z-index: 1;
```

Comment:
What will happen when we have elements which having different z-indexes? Ex: when applying this for modals or popovers? Will setting the z-index explicitly makes any issues?

Reply:
this z-index is for psudeo element, not the skeleton component.